python/lsst/meas/deblender/baseline.py

```diff
@@ -131,9 +134,11 @@ def __init__(self, footprints, maskedImages, psfs, psffwhms, log, filters=None,
             self.deblendedParents[self.filters[n]] = dp
 
         # Group the peaks in each color
+        self.peaks = []
```

Comment:
This commit was made on the main stack, but had not been merged yet. So I included it here so that it is one last conflict when we merge pybind11 with master (it is the same exact commit, I created this branch from the one I modified but have not merged yet).